libsolidity/interface/CompilerStack.cpp

```diff
@@ -122,6 +123,11 @@ bool CompilerStack::parse()
 	resolveImports();
 
 	bool noErrors = true;
+	SyntaxChecker syntaxChecker(m_errors);
+	for (Source const* source: m_sourceOrder)
+		if (!syntaxChecker.checkSyntax(*source->ast))
+			return false;
```

Comment:
I think it is fine continuing error checking, or is there some reason we should abort here?